### PR TITLE
Pick default project

### PR DESCRIPTION
**Pull request by isaacabraham**
@forki this is built on top of the **add-without-init** branch. It's totally the wrong way to do it, but if there's a way to identify any project in the "current" folder, then I think this approach would work. At least, testing locally it seems to do the right thing: you can go to an empty project, type `paket add Saturn` (assuming you have paket as a global tool) and everything will just work.

### CODE REVIEW

**Comment by isaacabraham** [comment]
@forki I think that this is good to go now. Behaviour is as follows on a completely clean project (no paket)

```cmd
dotnet Paket add Saturn
Paket version 6.0.0-beta2
Adding package 'Saturn'
Resolving dependency graph...
Created dependency graph (16 packages in total)
Extracted Paket.Restore.targets to: C:\Users\Isaac\Source\Repos\Samples\projectb\.paket\Paket.Restore.targets
Resolved package 'Saturn' to version 0.14.1
Total time taken: 5 seconds
```

I'm just going to move the Paket.Restore.Targets line to verbose as well, but otherwise I think that this is good to test.

File: src/Paket.Core/PackageManagement/AddProcess.fs [hunk 1]
```diff
@@ -25,11 +25,11 @@ let private add installToProjects addToProjectsF dependenciesFileName groupName
         traceWarnfn "%s contains package %O in group %O already." dependenciesFileName package groupName
     else
         let lockFileName = DependenciesFile.FindLockfile dependenciesFileName
-        let lockFile = ref None
+        let mutable lockFile = None
```
Comment (forki) [inline src/Paket.Core/PackageManagement/AddProcess.fs:28]: REF CELLS!!!

**Comment by forki** [comment]
the appveyor is red

**Comment by forki** [comment]
still red

**Comment by isaacabraham** [comment]
Mmm, need to look into that.

**Comment by isaacabraham** [comment]
Works on Travis but not AV.

**Comment by isaacabraham** [comment]
@forki does paket even need to support dotnet tools any more?

**Comment by isaacabraham** [comment]
Interestingly, just running the failing test command `paket add dotnet-fable --version 1.3.7 -t clitool --no-resolve` (from current live paket) also doesn't generate a lock file.

**Comment by isaacabraham** [comment]
Ok I think I know what the problem is!

**Comment by isaacabraham** [comment]
Green.

**Pull request merged**